components/notification-bar/notification-bar.config.yml

```diff
@@ -0,0 +1,29 @@
+context:
+  close_button: False
+  content: Generic notification.
+default: Default
+variants:
+  - name: Success
+    notes: Use the class `mzp-t-success` for a positive notification, to indicate a successful action. This example also features a close button.
```

Comment:
**question (potentially blocking):** Should the close buttons here be functional? Or are they only for demo purposes? Would be good to note if they are meant to be non-functional